### PR TITLE
stats: Update materialized views concurrently

### PR DESCRIPTION
**Pull request by mitjat**
Refreshing the materialized view takes ~5 min. Before this PR, the view was locked (even for read queries) during the refresh, making the related `/stats/tx_volume` endpoint unavailable for ~5min every ~1h, 1h being the refresh interval.

The changes (CREATE INDEX) have been applied to mainnet prod + staging indexers already.

Testing: Locally.

### CODE REVIEW

File: analyzer/queries/queries.go [hunk 1]
```diff
@@ -477,11 +477,11 @@ const (
       account_address = $3`
 
 	RefreshDailyTxVolume = `
-    REFRESH MATERIALIZED VIEW stats.daily_tx_volume
+    REFRESH MATERIALIZED VIEW CONCURRENTLY stats.daily_tx_volume
```
Comment (pro-wh) [inline analyzer/queries/queries.go:480]: :information_source: https://www.postgresql.org/docs/current/sql-refreshmaterializedview.html

> Without this option a refresh which affects a lot of rows will tend to use fewer resources and complete more quickly, but could block other connections which are trying to read from the materialized view.

**Review by pro-wh — APPROVED** [review]
index looks good too

File: analyzer/queries/queries.go (hunk 1 above)
Comment (ptrus) [inline analyzer/queries/queries.go:480]: Thanks!

From docs:

> This option may not be used when the materialized view is not already populated.

Does this mean that the first refresh on a fresh DB should be done non-concurrently?

**Comment by ptrus** [comment]
Might make sense opening an issue for future, to refactor this to not constantly recompute all data. I think the logic from `dailyActiveAccountsWorker` could be extracted and reused.

File: analyzer/queries/queries.go (hunk 1 above)
Comment (mitjat) [reply analyzer/queries/queries.go:480]: Good find. I tested now on a synthetic example and it's not the case (= we're ok), but I too found it hard to parse the docs. The view is populated at creation time. I think they're talking about the corner case of having created the view with `CREATE MATERIALIZED VIEW ... WITH NO DATA` [(docs)](https://www.postgresql.org/docs/current/sql-creatematerializedview.html) and then trying to do a `REFRESH ... CONCURRENTLY`.

**Comment by mitjat** [comment]
> Might make sense opening an issue for future, to refactor this to not constantly recompute all data. I think the logic from `dailyActiveAccountsWorker` could be extracted and reused.

Created https://app.clickup.com/t/866abpjhd

File: analyzer/queries/queries.go (hunk 1 above)
Comment (ptrus) [reply analyzer/queries/queries.go:480]: Good :+1: thanks for checking!
